File: android/src/main/java/com/valentingrigorean/arcgis_maps_flutter/map/PolygonControllerSink.java
package com.valentingrigorean.arcgis_maps_flutter.map;

import com.esri.arcgisruntime.symbology.SimpleLineSymbol;

public interface PolygonControllerSink extends GraphicControllerSink {

    void setFillColor(int color);

    void setStrokeColor(int color);

    void setStrokeWidth(float width);

    void setStrokeStyle(SimpleLineSymbol.Style style);
}

File: android/src/main/java/com/valentingrigorean/arcgis_maps_flutter/map/PolygonControllerSink.kt
package com.valentingrigorean.arcgis_maps_flutter.map

import com.esri.arcgisruntime.symbology.SimpleLineSymbol

interface PolygonControllerSink : GraphicControllerSink {
    fun setFillColor(color: Int)
    fun setStrokeColor(color: Int)
    fun setStrokeWidth(width: Float)
    fun setStrokeStyle(style: SimpleLineSymbol.Style?)
}